app/services/foreman_templates/template_importer.rb

```diff
@@ -131,5 +131,28 @@ def update_job_template(name, text)
       result = "  #{c_or_u} Template #{id_string}:#{name}"
       { :diff => diff, :status => template.save, :result => result }
     end
+
+    def purge!
+      # Available options:
+      #* verbose => Print extra information during the run [false]
+      #* not     => negate prefix / filter
+      #* kind    => Template kind
+      #* prefix  => The string all imported templates should begin with [Community ]
+      #* filter  => Import names matching this regex (case-insensitive; snippets are not filtered)
+
+      @verbose = ( ENV['verbose'] and ENV['verbose'] != 'false' ) ? true : false
```

Comment:
You can't use ENV in app/services, it won't exist. If you look at how the Rake task calls `import!` it passes the appropriate ENV variables in as options. Over-complicated I know, but it's for future stuff, I promise ;)